chapters/synchronous.tex

```diff
@@ -342,17 +342,18 @@ \section{Clock Constructors}\label{clock-constructors}
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
 \firstuse{Rational interval clock}.
-The first input argument, $\mathit{intervalCounter}$, is a clocked component expression (\cref{def:component-expression}) or a parameter expression of type \lstinline!Integer! with \lstinline!min = 0!.
+The first input argument, $\mathit{intervalCounter}$, is a clocked component expression (\cref{def:component-expression}), or a parameter expression of type \lstinline!Integer! with \lstinline!min = 0!.
 The optional second argument $\mathit{resolution}$ (defaults to 1) is a parameter expression of type \lstinline!Integer! with \lstinline!min = 1! and \lstinline!unit = "Hz"!.
 If $\mathit{intervalCounter}$ is a parameter expression with value zero, the period of the clock is derived by clock inference, see \cref{sub-clock-inferencing}.
+
+If $\mathit{intervalCounter}$ is a parameter expression greater than zero, the clock defines a periodic clock.
+If $\mathit{intervalCounter}$ is a clocked component expression it must greater than zero.
```

Comment:
Missing "be" in "it must greater than zero."

Reply:
Ok, added.